server/src/routes/api/sequence.mjs

```diff
@@ -0,0 +1,57 @@
+/*
+* Copyright 2021 Comcast Cable Communications Management, LLC
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+* http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*
+* SPDX-License-Identifier: Apache-2.0
+*/
+
+// HTTP-based API routes: State-Related
+
+'use strict';
+
+import { logger } from '../../logger.mjs';
+import {executeSequence} from '../../sequenceManagement.mjs';
+import * as commonErrors from '../../commonErrors.mjs';
+
+//Execute sequence events with respective delay values
+function sendSequence(req, res) {
+
+    try {
+        executeSequence(req,res);
```

Comment:
We want to keep all code that's NOT in /routes unaware of the req and res objects, which are Node/Express/HTTP/REST objects/concepts. So if you look at the other routes, they generally plug the passed in information (either from the body, url params, or url itself) and then pass these to the "helper function" (that's NOT in routes/).

So the code you have in server/src/sequenceManagement.mjs lines 29 through 37 (rummaging through the req and res objects should be here, and then the API signature for the executeSequence function should take those pieces of info (ws, userId, and seqevent).